Week1/homework/app.js

```diff
@@ -2,10 +2,130 @@
 {
   const bookTitles = [
     // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    'harry_potter_chamber_secrets',
+    'desert_solitaire',
+    'geek_love',
+    'gilead',
+    'disgrace',
+    'infinite_jest',
+    'lolita',
+    'maus',
+    'poems',
+    'things_fall_apart'
   ];
+  console.log(bookTitles);
```

Comment:
You can use this array to loop through the properties of your `bookDetails` object. See below.